crates/store/re_types/definitions/rerun/blueprint/archetypes/force_link.fbs

```diff
@@ -0,0 +1,15 @@
+namespace rerun.blueprint.archetypes;
+
+/// Aims to achieve a target distance between two nodes that are connected by an edge.
+struct ForceLink (
```

Comment:
Shouldn't that be named `ForceEdge` then? We don't use this "link" terminology anywhere else do we?